tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -272,14 +301,12 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
       "metadata": {
-        "colab": {},
         "colab_type": "code",
         "id": "IqBdl3uJN4r_",
-        "scrolled": true
+        "scrolled": true,
```

Comment:
Hoping that these changes are due to changes in Colab and not anything linting related. Otherwise, there will be back and forth.

Reply:
hmm, not sure. I looked at those too. It is strange since I didn't even run the notebook in Colab and I only opened it.

I tried to use the tool to clean the file like this:
`jupyter nbconvert --clear-output --inplace my_notebook.ipynb`
Then it generates something completely different from the current tfmot master. So I wonder what kind of cleaning tool do you use?

or, 

should I just manually editing the file in vim and make sure headers are the only change w.r.t. master tfmot? then next time if the change is massive, it will be tedious to edit them bits by bits.

Reply:
Let's avoid manually editing.

I opened things up on Colab and everything generally seems ok. I think it's fine for now and we can keep it in the back of our heads.

The flow I usually use is modifying things under colab.sandbox.google.com and then going to File > Save.

Reply:
yea, I do the same in Colab. 
Thanks.